tests/subsys/llext/simple/src/test_llext_simple.c

```diff
@@ -268,6 +268,53 @@ static LLEXT_CONST uint8_t multi_file_ext[] ELF_ALIGN = {
 LLEXT_LOAD_UNLOAD(multi_file, true, NULL)
 #endif
 
+static LLEXT_CONST uint8_t export_dependent_ext[] ELF_ALIGN = {
+	#include "export_dependent.inc"
+};
+
+ZTEST(llext, test_inter_ext)
+{
+	void *dependency_buf = k_heap_alloc(&tllext_heap, sizeof(hello_world_ext), K_NO_WAIT);
+
+	zassert_not_null(dependency_buf);
+
+	void *dependent_buf = k_heap_alloc(&tllext_heap, sizeof(export_dependent_ext), K_NO_WAIT);
+
+	zassert_not_null(dependent_buf);
+
+	memcpy(dependency_buf, hello_world_ext, sizeof(hello_world_ext));
+	memcpy(dependent_buf, export_dependent_ext, sizeof(export_dependent_ext));
+
+	struct llext_buf_loader buf_loader_dependency =
+		LLEXT_BUF_LOADER(dependency_buf, sizeof(hello_world_ext));
+	struct llext_buf_loader buf_loader_dependent =
+		LLEXT_BUF_LOADER(dependent_buf, sizeof(export_dependent_ext));
+	struct llext_loader *loader_dependency = &buf_loader_dependency.loader;
+	struct llext_loader *loader_dependent = &buf_loader_dependent.loader;
+	const struct llext_load_param ldr_parm = LLEXT_LOAD_PARAM_DEFAULT;
+	struct llext *ext_dependency = NULL, *ext_dependent = NULL;
+	int ret = llext_load(loader_dependency, "inter_ext_dependency",
+			     &ext_dependency, &ldr_parm);
+
+	zassert_ok(ret, "dependency load should succeed");
+
+	ret = llext_load(loader_dependent, "export_dependent", &ext_dependent, &ldr_parm);
+
+	zassert_ok(ret, "dependent load should succeed");
+
+	void (*test_entry_fn)() = llext_find_sym(&ext_dependent->exp_tab, "test_dependent");
+
+	zassert_not_null(test_entry_fn, "test_entry should be an exported symbol");
```

Comment:
`test_entry` in the message is really `test_dependent`

Reply:
yeah, there's one common entry point, so I thought it shouldn't matter